includes/discovery/sensors/bitrate/terra-sdi410c.inc.php

```diff
@@ -26,10 +26,12 @@
 $divisor = 1;
 $multiplier = 1000;
 
-if (is_array($pre_cache['sdi410cstatus'])) {
+$oids = SnmpQuery::cache()->hideMib()->walk('TERRA-sdi410C-MIB::sdi410cstatus')->table(1);
```

Comment:
This is only used here, caching is a waste of ram.

Reply:
done